zebra-test/src/transcript.rs

```diff
@@ -15,19 +15,25 @@ use tower::{Service, ServiceExt};
 
 type Error = Box<dyn std::error::Error + Send + Sync + 'static>;
 
+/// An error-checking function: is the value an expected error?
 pub type ErrorChecker = fn(Option<Error>) -> Result<(), Error>;
 
+/// An expected error in a transcript.
 #[derive(Debug, Clone)]
 pub enum TransError {
```

Comment:
I think this might be very nit-picky, so feel free to ignore this. Would it make sense to rename `TransError` into `TranscriptError` so it's less ambiguous? If so, it could even be in a separate PR to avoid increasing this PR's scope :man_shrugging:

Here are the changes that I think would be necessary:
```
for file in zebra-consensus/src/{block,chain}/tests.rs zebra-state/tests/basic.rs zebra-test/{src,tests}/transcript.rs; do
    sed -i "$file" -e 's/\<TransError\>/TranscriptError/g'
done
cargo fmt
```

Reply:
I would like `TranscriptError`

Reply:
Changed to ExpectedTranscriptError in d767a914

(It's not the actual error from a running transcript, it's a match rule for the errors expected by a transcript.)